tests/unit/test_templatetags.py

```diff
@@ -43,6 +44,22 @@ def test_get_translations(self, languages):
         language_codes = [l.code for l in translations.keys()]
         assert language_codes[0] == 'en'
         assert language_codes[1] == 'es'
-        assert language_codes[2] == 'fr' 
+        assert language_codes[2] == 'fr'
         assert language_codes[3] == 'de'
         assert language_codes[4] == 'nl'
+
+
+@pytest.mark.django_db
+class TestTranslationWagtailAdminTags(object):
+
+    def test_get_canonical_pages_for_delete(self, languages):
+        site = SiteFactory()
+        page = TranslatablePageFactory.build(language=languages[0])
+        site.root_page.add_child(instance=page)
```

Comment:
Well, will there be any extra benefit over the current approach?

Actually, at first, I did think of using `create_site_tree`. But `create_site_tree` function creates some additional subpages which I thought unnecessary in this particular case.